tests/test_project.py

```diff
@@ -250,6 +250,30 @@ def test_status_performance(self):
         assert time < 10
 
 
+class TestProjectStatusNoEligibleOperations(TestProjectBase):
+    class Project(FlowProject):
+        pass
+
+    @Project.operation
+    @Project.post(lambda job: True)
+    def foo(job):
+        pass
+
+    project_class = Project
+
+    def mock_project(self):
+        project = self.project_class.get_project(root=self._tmp_dir.name)
+        project.open_job(dict(i=0)).init()
+        return project
+
+    def test_status_no_eligible_operations(self):
+        """Test printing the project status when no operations are eligible."""
+        project = self.mock_project()
+        with redirect_stdout(StringIO()):
+            with redirect_stderr(StringIO()):
+                project.print_status()
```

Comment:
```suggestion
        with redirect_stdout(StringIO()),  redirect_stderr(StringIO()):
            project.print_status()
```

Reply:
I considered this but aimed for consistency with the existing test code instead. I'm going to leave this alone, if that's alright.

Reply:
I think a reasonable future change would be to add a context manager like `with silence_stdout_and_stderr()` that does both.

Reply:
Yeah that's fine. No reason to bother with two separate StringIO objects in that function either since we're just discarding the output anyway.